festim/mesh/mesh_1d.py

```diff
@@ -11,10 +11,10 @@ class Mesh1D(F.Mesh):
     1D Mesh
 
     Args:
-        vertices (list): the mesh x-coordinates (m)
+        vertices (list or np.array): the mesh x-coordinates (m)
```

Comment:
The correct type is `np.ndarray`

```suggestion
        vertices (list or np.ndarray): the mesh x-coordinates (m)
```